### DISCUSSION TITLE
Profile pictures don't go under barline on profile pages

### DISCUSSION THREAD

**Discussion by hwittenborn** [community/community#7931] [Other Feature Feedback, Questions, & Ideas]
The profile pictures on people's homepages doesn't go under the line like other elements on the page do (look at the pinned repositories for an example):

![image](https://user-images.githubusercontent.com/74838472/142712109-6c1008c3-30e6-4919-b8af-16e93a1e0435.png)

It isn't looking very nice to me, and I think it should instead go under the line like the other items to make it all look more uniform.

**Comment by NikhilMath** [comment]
if you are using google chrome, do a hard cache reset by doing Control (command) + shift + R?
